### PR TITLE
Avoid connection timeout of "inspec version"

### PR DESCRIPTION
**Pull request by makotots**
Fixes https://github.com/chef/inspec/issues/1537 
Signed-off-by: Makoto Nozaki <makoto.nozaki@twosigma.com>

### CODE REVIEW

**Comment by makotots** [comment]
Note: The next line, `http.read_timeout = 0.5` could be moved up to the same level for consistency.

File: lib/utils/latest_version.rb [hunk 1]
```diff
@@ -8,7 +8,8 @@ class LatestInSpecVersion
   # fetches the latest version from rubygems server
   def latest
     uri = URI('https://rubygems.org/api/v1/gems/inspec.json')
-    res = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https') {|http|
+    res = Net::HTTP.start(uri.host, uri.port, use_ssl: uri.scheme == 'https',
+                          open_timeout: 0.5) {|http|
       http.read_timeout = 0.5
```
Comment (adamleff) [inline lib/utils/latest_version.rb]: I agree with your comment that this should be moved up to the `#start` opts for consistency. Thank you for suggesting it!

**Review by adamleff — CHANGES_REQUESTED** [review]
@makotots thank you so much for this PR! I agree that keeping the definition of the options at the same level for consistency is the right thing to do. Would you mind pushing up a new commit with that change? Then I'll be happy to get this approved.

**Review by chris-rock — APPROVED** [review]
Thank you @makotots and @adamleff This is a great improvement.

**Review by adamleff — APPROVED** [review]

**Comment by adamleff** [comment]
Great work, @makotots! Thank you for your contribution!

**Pull request merged**